src/titiler/extensions/titiler/extensions/viewer.py

```diff
@@ -36,6 +36,8 @@ def cog_viewer(request: Request):
                     ),
                     "info_endpoint": factory.url_for(request, "info"),
                     "statistics_endpoint": factory.url_for(request, "statistics"),
+                    "viewer_enabled": hasattr(factory, "add_viewer")
```

Comment:
so if the attribute is there but set to False what happens?

Reply:
If it is set to `False` the `Share Map` button is hidden in the webmap:
![image](https://github.com/user-attachments/assets/028ac4af-8eb4-4650-86b0-87329b1280d4)

The check for `getattr(factory, "add_viewer", False)` is on the next line but is not obvious with `black` formatting :smile: